### DISCUSSION TITLE
Plugins

### DISCUSSION THREAD

**Discussion by robotard** [wwmm/easyeffects#1259] [General]
Hey guys,

Previous to PipeWire, I used to have a script to install PulseEffects and it's plugins (as well as copying my presets across to the old ~./config location), but am a little uncertain if this would be the correct process now. 

For example, I think you guys are no longer using gstreamer with PipeWire (not even sure if I need it for anything else on Ubuntu such as Chrome)...

I was wondering which of the below (if any) are being or can be automated into a bash script similar to the one in my one in the link at the bottom.

```
sudo apt install gstreamer1.0-plugins-base \
	gstreamer1.0-plugins-good \
	gstreamer1.0-plugins-bad \
	gstreamer1.0-convolver-pulseeffects \
	gstreamer1.0-crystalizer-pulseeffects \
	gstreamer1.0-autogain-pulseeffects \
	lsp-plugins \
	calf-plugins \
	zam-plugins \
	rubberband-ladspa \
	liblilv-0-0 \
	mda-lv2 \
	libsndfile1 \
	libsamplerate0 \
	libebur128-1 -y

```
Prior to PipeWire, I wrote this script to install the old PulseEffects:

https://github.com/robotard/setup-ubuntu/blob/master/scripts/pulseeffects.sh

**Comment by wwmm** [comment]
We do not use GStreamer anymore. So you can remove its packages from your script.

**Reply by robotard** [reply]
Yes, saw that was replaced with pipewire now... 

Are they audio only? i.e. not used by chrome for video or anything?

I am rewriting that script purely for EasyEffects, but if it's used elsewhere, I'd like to move it to a correct script so as to manage all my reinstall scripts (I quite deliberately break things a lot in my engineering travels, so regularly reinstall from scratch - may be useful for your testing too ;) )

**Reply by wwmm** [reply]
>Are they audio only?

No. GStreamer has plugins for video processing too. But usually the applications that use them will have the corresponding GStreamer modules as dependency.
